net/openconnect/files/openconnect-wrapper

```diff
@@ -10,32 +10,4 @@ test -z "$1" && exit 1
 
 pwfile=$1
 shift
-
-pidfile=/var/run/ocwrap-$$.pid
-
-cleanup()
-{
-	if ! test -z "$pid";then
-		kill $pid
-		wait $pid
-	fi
-	exit 0
-}
-
-cleanup2()
-{
-	if ! test -z "$pid";then
-		kill -2 $pid
-		wait $pid
-	fi
-	exit 0
-}
-
-trap cleanup2 2
-trap cleanup 1 3 6 15
```

Comment:
Let's separate the pull request to two different things. One that you add the mtu and this one. I remember there was a reason this wrapper was added and trapped all this signals; removing it would need good justification and testing, which is orthogonal to the mtu option.